lib/src/generators/contract/__snapshots__/json-schema.spec.ts.snap

```diff
@@ -233,6 +236,8 @@ definitions:
             $ref: '#/definitions/Email'
           address:
             $ref: '#/definitions/Address'
+          createdAt:
+            type: string
```

Comment:
Shouldn't the changes have generated the `format` too?
```
type: string
format: date
```

Reply:
hey @alephao, this is the old implementation. We should not use it anymore. I'll create a ticket to clean this up.